robots/lwr_ft.urdf.xacro

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="lwr">
+
+  <xacro:arg name="use_gazebo" default="1"/>
+  <xacro:arg name="use_transmission" default="1"/>
+  <xacro:arg name="collision_simple" default="1"/>
+  <xacro:arg name="visual_simple" default="0"/>
+
+  <!-- KUKA LWR4 arm -->
+  <xacro:include filename="$(find kuka_lwr4_description)/robots/kuka_lwr_arm.urdf.xml"/>
+
+  <!-- force torque sensor -->
+  <xacro:include filename="$(find kuka_lwr4_description)/defs/force_torque_sensor_defs.xml"/>
```

Comment:
You mean by passing a parameter with the name of the sensor or end-effector?

Reply:
That particulary included ``force_torque_sensor`` - is it the ATI that Joao used previously (wiping) or the currently used one or a generic one? E.g. if it is modeled after the ATI something like ati_modelnumber_force_torque_sensor would be good. If it is totally generic, lets leave it as is.

Reply:
I don't know about the type of force torque sensor. The `force_torque_sensor_defs.xml` is in the repo since the initial commit and there is no information about the FT type within the file.
Maybe @VladimirIvan can say more about the type and model of the FT sensor?